runtime-modules/working-team/src/checks.rs

```diff
@@ -0,0 +1,160 @@
+use crate::{Instance, JobOpening, JobOpeningType, TeamWorkerId, Trait};
+
+use super::Error;
+use frame_support::dispatch::{DispatchError, DispatchResult};
+use frame_support::{ensure, StorageMap, StorageValue};
+use sp_std::collections::btree_set::BTreeSet;
+use system::{ensure_root, ensure_signed};
+
+use crate::types::ApplicationInfo;
+
+// Check opening: verifies opening description length.
+pub(crate) fn ensure_opening_description_is_valid<T: Trait<I>, I: Instance>(
```

Comment:
We dont need to limit this, as we are no longer storing this.

Reply:
Removed.